userguide/aws/how-tos/add-a-certificate.adoc

```diff
@@ -0,0 +1,157 @@
+
+If you own a domain, your domain provider can serve as your CA. If you don't have a domain, you can purchase one for very low cost from a domain provider such as Godaddy. When you obtain a domain you need to prove your identity, so your domain provider can trust you. To demonstrate this trust you will need a CNAME.
+
+== Preliminary concepts
+
+=== CNAMEs
+
+A CNAME, or *Canonical Name*, is an identifier that links your CA to the resources that you are protecting with your certificate. You will be protecting public access to your instances of Kaui and Kill Bill through your load balancer, identified by its DNS name. The CNAME will link your domain to the load balancer. If your domain is `mydomain.com` your CNAME might look like `kaui.mydomain.com`. This will direct browsers to your verified domain, and then, through your CNAME, to your AWS resources.
+
+The CNAME is part of your domain's DNS profile, and your domain provider should have instructions for setting it up. These instructions should ask you to enter two strings called `Name` and `Value`. These strings will be obtained during the certificate generation process.
+
+=== Protocol and Port Setup
+
+Your AWS security group by default provides rules giving access to Kaui and Kill Bill using the HTTP protocol and a specified port number. For HTTPS you will need additional rules with the new protocol and a new port number.
+
+=== Validation
+
+When you create a new certificate you will need to validate your certificate. This identifies the trust hierarchy for the certificate. You may need to setup a temporary CNAME for this purpose, and you may need to provide temporary access via an extra port in your security group.
+
+== Obtaining a Certificate
+
+There are many ways to obtain a certificate that you can use to protect a particular resource. Here we will describe two that may be of use in particular Kill Bill scenarios running on AWS. These are:
+
+1. Using the AWS Certificate Manager
+2. Using the free CA *Let's Encrypt*
+
+== Using the AWS Certificate Manager (ACM)
+
+The AWS Certificate Manager (ACM) offers a straightforward way to provide SSL/TLS protection for AWS resources such as load balancers. This is recommended when using the multi-tier or cloud formation installation options. There is no extra cost for using ACM certificates. 
+
+=== 1. Request a Certificate
+
+If you are on a page that is asking for your certificate, click on *Request a new Certificate from ACM*. Otherwise, select ACM from the Services menu. You will be taken to the main page of the ACM. Initially this will probably show that you have no certificates. Click *Request a Certificate*.
+
+The next page will give you the option to import a certificate, or to request a public or private certificate. Choose *private*, then click again on *Request a Certificate*. This will take you to the *Request a Certificate* page. You are initially at *Step 1: Add Domain Names*.
```

Comment:
```suggestion
+
The next page will give you the option to import a certificate, or to request a public or private certificate. 
+
. Choose *private*, then click again on *Request a Certificate*. This will take you to the *Request a Certificate* page. You are initially at *Step 1: Add Domain Names*.
```

Reply:
@jdmooney - I found myself wanting to see more screenshots in this section, especially in the beginning. For example, when it says, "Enter your domain name...", I am not sure where to enter it. 

I know these AWS screens will likely change, but I think it's better to do a lot of handholding here. We can always revise/revisit later.

Reply:
I see now where to enter the domain name. Something like "Enter the domain name in the **Fully Qualified Domain Name** field (shown above)."